### DISCUSSION TITLE
How to manually configure fonts of oh-my-posh in linux(fedora)

### DISCUSSION THREAD

**Discussion by netcore-jroger** [JanDeDobbeleer/oh-my-posh#3635] [General]
In linux, where is the config file of oh-my-posh? I need to change font manually. my computer are without network.

**Comment by netcore-jroger** [comment]
I tried to modify the font of the terminal, but the icon is still garbled.

**Reply by JanDeDobbeleer** [reply]
@netcore-jroger depends on how you initialised, you can fond out using `echo $PSOH_THEME`. But if that outputs none, you're using the default config and you can export it using the following [guide](https://ohmyposh.dev/docs/installation/customize#adjust-a-theme).

**Reply by netcore-jroger** [reply]
@JanDeDobbeleer  I manually installed FireCode/CascadiaCode/JetBrainsMono fonts and restarted the terminal, and the characters are no longer garbled. Thank you for your reply.

It would be better if the oh-my-posh install command can support the path of the local font compression package file

**Reply by JanDeDobbeleer** [reply]
@netcore-jroger you need allow font install to use a local font zip?

**Reply by netcore-jroger** [reply]
A good solution when there is no network. pyenv has a similar function, put the source file in the cache directory, it will be automatically installed from the cache directory.

**Reply by JanDeDobbeleer** [reply]
@netcore-jroger but in that case you can install a font manually entirely, no? We basically do the same for you. Nonetheless, maybe it's a good addition.

**Reply by netcore-jroger** [reply]
Yes, a nice addition.  A lot of command line operations have been reduced.  It can better reflect the flexibility of this tool.

**Reply by JanDeDobbeleer** [reply]
@netcore-jroger https://github.com/JanDeDobbeleer/oh-my-posh/pull/3648
